modules/patterns/pages/centralize-pipeline-definitions.adoc

```diff
@@ -0,0 +1,146 @@
+= Pattern: Centralizing Tekton Pipeline Definitions in Your Repository
+
+This pattern describes how to extract a common Tekton `Pipeline` definition into a local file within your application's Git repository (e.g., `.tekton/build-pipeline.yaml`). Your `PipelineRun` files can then reference this local pipeline, promoting consistency, simplifying maintenance, and enabling version control of the pipeline logic alongside your application code.
+
+This approach is particularly beneficial when you need to customize a standard pipeline or ensure that all build triggers (e.g., push, pull request) use the exact same underlying pipeline tasks and structure.
+
+.Use Cases
+* Customizing a standard {ProductName} build pipeline (e.g., `docker-build-multi-platform-oci-ta`) with additional tasks or modified parameters.
```

Comment:
I suggest dropping this use case.

```suggestion
* Customizing a standard {ProductName} build pipeline (e.g., `docker-build-multi-platform-oci-ta`) with additional tasks or modified parameters.
```

It's true that this pattern may *facilitate* easier customization, but the pattern doesn't add tasks or modified parameters on its own.

The other three use cases really resonated with me.

Reply:
I think that this is better represented as a pattern for the managing pipeline changes pattern. I will remove it from this.